### DISCUSSION TITLE
Cyfrin updraft website is not working and responding

### DISCUSSION THREAD

**Discussion by Snach13** [Cyfrin/foundry-full-course-cu#2513] [General]
the website keeps loading doesnt showing the content I dont whats the issue with it I tried with different browsers but same result

**Comment by EngrPips** [comment]
Hello @Snach13, are you still having this issue?

**Reply by Snach13** [reply]
@EngrPips Yes I'm still facing the issue

**Reply by EngrPips** [reply]
Please join the Cyfrin official discord https://discord.gg/PDeKGNS5 and report your issue there. You will get a quick response on there.

**Comment by akkisec** [comment]
website is not working

**Reply by EngrPips** [reply]
Seems to be loading slowly for me too, I have called the attention of the Cyfrin team to it.

**Comment by 08139984630** [comment]
Please I'm unable to perform my transactions 
![Uploading Screenshot_20241226-083037.png…]()

**Reply by EngrPips** [reply]
Hello @08139984630  the image you shared is not loading for me.

**Comment by 08139984630** [comment]
@EngrPips I can only send link sir
![Uploading Screenshot_20241226-083037.png…]()

**Reply by EngrPips** [reply]
What transaction are you trying to perform?

**Reply by 08139984630** [reply]
I'm trying to perform a sepolia eth testnet transaction sir

**Reply by EngrPips** [reply]
What sepolia transaction? Are you trying to deploy a contract, or are you trying to get sepolia ether from a faucet or what are you trying to do? Please be specific.

**Reply by 08139984630** [reply]
I'm trying to get sepolia ether from a faucet

**Reply by EngrPips** [reply]
Try this https://cloud.google.com/application/web3/faucet/ethereum/sepolia If it did not work on the first try then try again after 15 minutes and it should work.

**Reply by 08139984630** [reply]
Ok sir thanks
Let me try

**Reply by EngrPips** [reply]
You are welcome, sir.

**Reply by 08139984630** [reply]
Wow it's actually worked

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.

**Comment by 08139984630** [comment]
Hello sir I don't understand the hash, block, Blockchain and distribution aspect

**Reply by EngrPips** [reply]
I will advise you not to bother yourself with all of those now and keep going with the course. They will get to make sense at a later time.

**Reply by 08139984630** [reply]
Ok sir 
I already stopped cos my brain is getting hot but I will continue back

**Reply by EngrPips** [reply]
Good that you stopped. Somethings won't make sense until a later time.
